doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -0,0 +1,986 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "result = np.asarray(result).T\n",
+    "\n",
+    "labels = ['all', 'x-axis', 'y-axis', 'z-axis']\n",
+    "for col, label in zip(result, labels):\n",
+    "    plt.plot(col, label=label)\n",
+    "plt.legend()\n",
+    "plt.ylabel('Radius of gyration (Å)')\n",
+    "plt.xlabel('Frame');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parallelization with dask\n",
+    "\n",
+    "[Dask](https://docs.dask.org/en/latest/) is a flexible library for parallel computing in Python. It provides advanced parallelism for analytics that has been integrated or utilized in many scientific softwares. It can be scaled from one single computer to a cluster of computers inside a HPC center.\n",
```

Comment:
```suggestion
    "[Dask](https://docs.dask.org/en/latest/) is a flexible library for parallel computing in Python. It provides advanced parallelism for analytics and has been integrated or utilized in many scientific softwares. It can be scaled from one single computer to a cluster of computers inside a HPC center.\n",
```